src/styles/cloudmeter/_verticalNav.scss

```diff
@@ -0,0 +1,15 @@
+
+.nav-pf-vertical {
+  display: none;
+}
+
+.collapsed-nav .cloudmeter-content,
+.cloudmeter-content {
```

Comment:
Initially, this was just to get it up and working with a masthead only, messy for sure. But it does relate to the off-to-the-side conversation about handling a masthead only PF-React component while still needing some form of action in the hamburger menu. ( there should be a media query update to this since this comment )